File: 138/Easy.py
from math import sqrt
m1, x1, y1 = map(float, raw_input().split(" "))
m2, x2, y2 = map(float, raw_input().split(" "))
dX, dY = x1 - x2, y1 - y2
dist = sqrt(dX**2 + dY**2)
print (m1*m2)/(dist**2)

File: 146/Easy.clj
(require '[clojure.string :as str])
(defn easy-146 []
  (let [n-r (map read-string (str/split (read-line) #" "))]
    (let [n (first n-r)]
      (let [r (last n-r)]
        (prn (format "%.3f"
          (* 2 n r (Math/sin (/ Math/PI n)))))))))

(easy-146)
